src/rgw/radosgw-admin/orphan.cc

```diff
@@ -1,6 +1,12 @@
+
+/*
+ * Copyright (C) 2024 IBM 
+*/
+
```

Comment:
By placing this block ahead of the editor instructions, emacs is no longer indenting appropriately. We need to flip the order of the two blocks.

Reply:
@chardan ☝🏼

Is that something you have bandwidth to fix with a follow-on PR?

Reply:
I hit it with radosgw_admin.cc .

Reply:
My apologies!

https://github.com/chardan/ceph/pull/new/wip-jfw-rgw-fix-editor-mode